umm-spec-lib/src/cmr/umm_spec/metadata_specification.clj

```diff
@@ -1,17 +1,24 @@
 (ns cmr.umm-spec.metadata-specification
   "Functions related to the MetadataSpecification node common to many of the umm
    models. Eventually all models will include this node, so a common set of
-   functions is needed."
+   functions is needed.
+   "
```

Comment:
You can end the comment on the same line, newline isn't necessary here

Reply:
Sorry, I merged already, will add to my next change.